roboflow/config.py

```diff
@@ -1,23 +1,67 @@
 import os
+import json
 
-CLASSIFICATION_MODEL = os.getenv("CLASSIFICATION_MODEL", default="ClassificationModel")
+
+conf_location = os.getenv(
+    "ROBOFLOW_CONFIG_DIR", default=os.getenv("HOME") + "/.config/roboflow/config.json"
+)
+
+# read from json file if exists
+if os.path.exists(conf_location):
+    # read json from file
+    with open(conf_location) as f:
+        config = json.load(f)
+else:
+    config = {}
+
+
+def get_var_conditional(key, default):
+
+    if os.getenv(key) != None:
+        return os.getenv(key)
+    elif key in config.keys():
+        return config[key]
+    else:
+        return default
```

Comment:
could we avoid the complexity of 2 config files by reading the response of the login json and then just setting the relevant ENV variable(s) (rather than writing it to a file)? the downside here would be that each new terminal instance would require a new login. are there security concerns with leaving the user's API keys on the filesystem?